meta-ibm/recipes-phosphor/chassis/mux-workarounds/mux-workarounds.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Perform workarounds for I2C Muxes
+Wants=sysinit.target
+After=sysinit.target
+
+[Service]
+ExecStart=/usr/bin/mux-workarounds.sh
+SyslogIdentifier=mux-workarounds
```

Comment:
nit: This isn't needed anymore since you now have the full path to the script. Not a big deal though.